### PR TITLE
improve main Readme

### PR DESCRIPTION
**Pull request by kristianmandrup**
Should make it more clear what the project can do for people who find the github repo and not the published packages ;)

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -16,6 +18,71 @@ npm install @gql2ts/from-query
...
+This project can either:
+
+- Take a schema and generate type definitions from it
+- Take a schema & a query and generate type definitions from it
+
+### Generate from schema
+
+Takes a type schema
+
+```txt
```
Comment (brettjurgens) [inline Readme.md]: we can use graphql as the language here

File: Readme.md [hunk 2]
```diff
@@ -16,6 +18,71 @@ npm install @gql2ts/from-query
...
+type Query {
+  thing: String!
+  anotherThing: Boolean!
+}
+```
+
+Generates TypeScript interface
+
+```txt
```
Comment (brettjurgens) [inline Readme.md]: likewise, we can use `ts` here

File: Readme.md [hunk 3]
```diff
@@ -16,6 +18,71 @@ npm install @gql2ts/from-query
...
+interface GetThing {
+  thing: string;
+}
+```
+
+Note that both `flow` and `typescript` generator outputs are supported via separate packages:
+
+- `language-flow` - set of defaults for running `gql2ts` with Flow as your target.
```
Comment (brettjurgens) [inline Readme.md]: should we match the npm names here? (i.e. `@gql2ts/*`)

File: Readme.md (hunk 1 above)
Comment (kristianmandrup) [reply Readme.md]: Yes, I just realised ;) Thanks

**Pull request merged**